src/redux/explorer.js

```diff
@@ -301,26 +307,22 @@ export const emitAssetInfoRequest = () => (dispatch, getState) => {
 };
 
 export const emitChartsRequest = (
-  assetAddress,
+  assetAddress = null,
```

Comment:
do we need this `= null`?

Reply:
hey just putting this here preemptively when someday I daydream of typescripting this and it'll yell at me that I didn't set a default value for an optional parameter (this is for the "get ALL assets charts when you hit All" flow)

Reply:
hey I removed this and explicitly set the asset codes in CoinDivider now!